web/packages/teleport/src/Bots/Add/GitHubActions/ConfigureBot.tsx

```diff
@@ -129,7 +129,13 @@ export function ConfigureBot({ nextStep, prevStep }: FlowStepProps) {
                     botName: e.target.value,
                   })
                 }
+                css={`
+                  margin-bottom: 0;
+                `}
               />
+              <Text color="text.slightlyMuted" fontSize="small">
+                Allowed characters: A-z, 0-9, and -.+
+              </Text>
```

Comment:
maybe i'm blind, but i didn't see that the `requireValidBotName` had a regex check for these chars?

Reply:
Yeah, I was just checking for whitespaces and letting the backend do the full validation. But it makes sense to add it there, updated in https://github.com/gravitational/teleport/pull/37443/commits/a598bc2433f7c6a8d466faeb0f23c22718f9aa6b